bedrock/firefox/views.py

```diff
@@ -541,7 +541,17 @@ def get_template_names(self):
             else:
                 template = 'firefox/whatsnew/index.html'
         elif channel == 'beta':
-            template = 'firefox/whatsnew/index.html'
+            if version.startswith('72.'):
```

Comment:
Not a blocker: you could probably have one less `else` conditional here by saying

```suggestion
            if version.startswith('72.') and locale in ['en-US', 'en-CA', 'en-GB']:
```

Reply:
Doh, nevermind I read this wrong!